.gitignore

```diff
@@ -25,7 +25,7 @@ docs
 *debug.log*
 yarn.lock
 .DS_Store
-.jhipster
+/.jhipster
```

Comment:
does jhipster create a .jhipster folder at system root level?

Reply:
In a generated app yes, to store the entity json definition file. In the blueprint source code, might be no.